mllib/src/main/scala/org/apache/spark/ml/clustering/LDA.scala

```diff
@@ -758,6 +816,10 @@ class LDA @Since("1.6.0") (
   @Since("1.6.0")
   def setOptimizeDocConcentration(value: Boolean): this.type = set(optimizeDocConcentration, value)
 
+  /** @group expertSetParam */
+  @Since("2.0.0")
+  def setKeepLastCheckpoint(value: Boolean): this.type = set(keepLastCheckpoint, value)
+
```

Comment:
I don't think so.  Once there is a model, the decision about keeping the last checkpoint has already been made.  Users can manage the checkpoint via the deletion method though.